src/util/types.ts

```diff
@@ -1109,6 +1109,8 @@ export interface LabelOption extends TextCommonOption {
     // formatter?: string | ((params: CallbackDataParams) => string)
 
     rich?: Dictionary<TextCommonOption>
+    lineOverflow?: 'truncate'
```

Comment:
I'm not sure if the default value should be `'truncate'` since the document takes `'none'` for default values. @pissang

Reply:
Just take this line of code, it's not to specify the default value but to define the type of `lineOverflow`.